src/openrct2-ui/windows/ClearScenery.cpp

```diff
@@ -224,12 +224,80 @@ namespace OpenRCT2::Ui::Windows
             return ClearAction(range, itemsToClear);
         }
 
+        int8_t ToolUpdateClearLandPaint(const ScreenCoordsXY& screenPos)
```

Comment:
This function was previously shared with the land tool, which is how it was missed.